### PR TITLE
Update about-us.md

### PR DESCRIPTION
**Pull request by adorayi**
Adding link to the roadmap in about us page EN

# Summary | Résumé

> 1-3 sentence description of the changed you're proposing, including a link to
> a GitHub Issue # or Trello card if applicable.
Where the roadmap is referenced in the about us page, adding a link to the roadmap page.
https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/960
---

# Test instructions | Instructions pour tester la modification

> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
> will help a developer test things out without too much detective work. Also,
> include any environmental setup steps that aren't in the normal README steps
> and/or any time-based elements that this requires.

Test the link is working in EN and FR
---

> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
> modification. Elles aideront les développeurs à faire des tests sans avoir à
> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
> de l’environnement qui ne font pas partie des étapes normales dans le fichier
> README et tout élément temporel requis.

### CODE REVIEW

File: src/fr/a-propos.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Le [Service numérique canadien](https://numerique.canada.ca/) crée et entretie
 
 ## Notre collaboration avec le système de conception de Canada.ca
 
-Système de design GC intègre les éléments obligatoires de Canada.ca. Nous collaborons avec le Bureau de la transformation numérique (BTN) au sein du Service numérique canadien afin d'évaluer le système de design et d'ajouter des éléments à notre feuille de route. Le BTN continue à offrir ses conseils et son soutien aux praticiens et praticiennes du Web et du numérique au sein du GC.
+Système de design GC intègre les éléments obligatoires de Canada.ca. Nous collaborons avec le Bureau de la transformation numérique (BTN) au sein du Service numérique canadien afin d'évaluer le système de design et d'ajouter des éléments à notre <gcds-link href="{{ links.feuille-de-route }}">feuille de route</gcds-link>. Le BTN continue à offrir ses conseils et son soutien aux praticiens et praticiennes du Web et du numérique au sein du GC.
```
Comment (melaniebmn) [inline src/fr/a-propos.md:23]:
```suggestion
Système de design GC intègre les éléments obligatoires de Canada.ca. Nous collaborons avec le Bureau de la transformation numérique (BTN) au sein du Service numérique canadien afin d'évaluer le système de design et d'ajouter des éléments à notre <gcds-link href="{{ links.roadmap }}">feuille de route</gcds-link>. Le BTN continue à offrir ses conseils et son soutien aux praticiens et praticiennes du Web et du numérique au sein du GC.
```

**Review by melaniebmn — CHANGES_REQUESTED** [review]
If you accept my suggestion, your link should also work in french @adorayi

File: src/fr/a-propos.md (hunk 1 above)
Comment (daine) [inline src/fr/a-propos.md:23]: French link doesn't work - it needs the variable `links.roadmap`

```suggestion
Système de design GC intègre les éléments obligatoires de Canada.ca. Nous collaborons avec le Bureau de la transformation numérique (BTN) au sein du Service numérique canadien afin d'évaluer le système de design et d'ajouter des éléments à notre <gcds-link href="{{ links.roadmap }}">feuille de route</gcds-link>. Le BTN continue à offrir ses conseils et son soutien aux praticiens et praticiennes du Web et du numérique au sein du GC.
```

**Review by daine — CHANGES_REQUESTED** [review]
@adorayi just one small change request for the link that's not working

File: src/fr/a-propos.md (hunk 1 above)
Comment (daine) [reply src/fr/a-propos.md:23]: Resolving as it's exactly the same suggestion @melaniebmn suggested above

**Review by daine — APPROVED** [review]
Not sure what's happening with amplify but I redeployed your branch here: https://adorayi-patch-linktoroadmap.d35vdwuoev573o.amplifyapp.com/fr/a-propos/

and I can confirm that it works

Screenshot 2024-09-11 at 3 35 42 PM


Looks good to me!

**Review by melaniebmn — APPROVED** [review]
LGTM - ship it!

**Pull request merged**